spec/requests/conversion_hosts_spec.rb

```diff
@@ -1,7 +1,11 @@
 describe "ConversionHosts API" do
+  before(:each) do
```

Comment:
Minor, but the :each is the default, and thus not necessary.

Reply:
Ok, removed.